### PR TITLE
Add new pages

### PR DESCRIPTION
**Pull request by hnucxr**
1.Add pages "alt" value.
2.Added Arabic page, Dutch page, Spanish page.

### CODE REVIEW

File: index-zh.html [hunk 1]
```diff
@@ -91,11 +91,14 @@
                           
                             <!-- Languages -->
                             <li>
-                                <a href="#" class="mn-has-sub"><i class="fa-solid fa-earth-americas"></i>英文 <i class="fa fa-angle-down"></i></a>
+                                <a href="#" class="mn-has-sub"><i class="fa-solid fa-earth-americas"></i>English <i class="fa fa-angle-down"></i></a>
```
Comment (saamerm) [inline index-zh.html]: This should still be : 英文

File: index-nl.html [hunk 2]
```diff
@@ -0,0 +1,498 @@
...
+                  <!-- Main Menu -->
+                  <div class="inner-nav desktop-nav">
+                      <ul class="clearlist scroll-nav local-scroll">
+                          <li class="active"><a href="#home">Thuis</a></li>
+                          <li><a href="#about">Over</a></li>
+                          <li><a href="#features">Functies</a></li>
+                          <li><a href="#screenshots">Schermafbeeldingen</a></li>                            
+                          <!-- Languages -->
+                          <li>
+                            <a href="#" class="mn-has-sub"><i class="fa-solid fa-earth-americas"></i>English <i class="fa fa-angle-down"></i></a>
```
Comment (saamerm) [inline index-nl.html]: This should say Nederlands

File: index-ge.html [hunk 3]
```diff
@@ -91,13 +91,16 @@
                         <li><a href="#screenshots">Screenshots</a></li>                            
                         <!-- Languages -->
                         <li>
-                            <a href="#" class="mn-has-sub"><i class="fa-solid fa-earth-americas"></i>Deutsche <i class="fa fa-angle-down"></i></a>
```
Comment (saamerm) [inline index-ge.html]: This should be Deutsche

File: index-fr.html [hunk 4]
```diff
@@ -91,13 +91,16 @@
                         <li><a href="#screenshots">Captures d'écran</a></li>                            
                          <!-- Languages -->
                          <li>
-                            <a href="#" class="mn-has-sub"><i class="fa-solid fa-earth-americas"></i>Français <i class="fa fa-angle-down"></i></a>
```
Comment (saamerm) [inline index-fr.html]: This should still be : Français

File: index-es.html [hunk 5]
```diff
@@ -0,0 +1,498 @@
...
+                  <!-- Main Menu -->
+                  <div class="inner-nav desktop-nav">
+                      <ul class="clearlist scroll-nav local-scroll">
+                          <li class="active"><a href="#home">Hogar</a></li>
+                          <li><a href="#about">Acerca de</a></li>
+                          <li><a href="#features">Características</a></li>
+                          <li><a href="#screenshots">Capturas de pantalla</a></li>                            
+                          <!-- Languages -->
+                          <li>
+                            <a href="#" class="mn-has-sub"><i class="fa-solid fa-earth-americas"></i>English <i class="fa fa-angle-down"></i></a>
```
Comment (saamerm) [inline index-es.html]: This should be Español

File: index-ar.html [hunk 6]
```diff
@@ -0,0 +1,496 @@
...
+                  <!-- Main Menu -->
+                  <div class="inner-nav desktop-nav">
+                      <ul class="clearlist scroll-nav local-scroll">
+                          <li class="active"><a href="#home">بيت</a></li>
+                          <li><a href="#about">عن</a></li>
+                          <li><a href="#features">سمات</a></li>
+                          <li><a href="#screenshots">لقطات الشاشة</a></li>                            
+                          <!-- Languages -->
+                          <li>
+                            <a href="#" class="mn-has-sub"><i class="fa-solid fa-earth-americas"></i>English <i class="fa fa-angle-down"></i></a>
```
Comment (saamerm) [inline index-ar.html]: This should be العربيه

**Review by saamerm — CHANGES_REQUESTED** [review]

**Review by hnucxr — COMMENTED** [review]
update language

**Review by saamerm — APPROVED** [review]

**Pull request merged**